rbi/core/class.rbi

```diff
@@ -69,6 +69,11 @@
 # obj--->OtherClass---------->(OtherClass)-----------...
 # ```
 class Class < Module
+  extend T::Generic
+  has_attached_class!(:out)
```

Comment:
The hard part of this change is just two lines. (before `has_attached_class!`, this change involved a lot of work and scoping).